### PR TITLE
Add in APIv4 Actions documentation

### PR DESCRIPTION
**Pull request by seamuslee001**

### CODE REVIEW

File: docs/api/v4/actions.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## delete
+
+Delete one or more records. (Note: Requires an explicit `id`. Note: if you want to skip the 'recycle bin' for entities that support undelete (e.g. contacts) you should set `$param['skip_undelete'] => 1);`
+
+## get
+
+Search for records
+
+## getfields
```
Comment (MikeyMJCO) [inline docs/api/v4/actions.md]:
```suggestion
## GetFields
```

File: docs/api/v4/actions.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Fetch entity metadata, i.e. the list of fields supported by the entity. There is now an option that you can pass in for get fields which is `loadOptions`. This is the equivilant of the apiv3 getoptions API call. When used for any field that specifies a pseudoconstant it will return the relevant options in an options key. You can also pass in an option `IncludeCustom` which will specifiy whether to include the relevant custom fields for that entity or not.
+
+## replace
```
Comment (MikeyMJCO) [inline docs/api/v4/actions.md]:
```suggestion
## Replace
```

File: docs/api/v4/actions.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Replace an old set of records with a new or modified set of records. (For example, replace the set of "Phone" numbers with a different set of "Phone" numbers.).
+
+Warning - REPLACE includes an implicit delete - use with care & test well before using in productions
```
Comment (MikeyMJCO) [inline docs/api/v4/actions.md]:
```suggestion
!!! warning
    Replace includes an implicit delete action - use with care & test well before using in production.
```

File: docs/api/v4/usage.md [hunk 4]
```diff
@@ -11,6 +11,9 @@ Canonically, an API call is processed by the API kernel.  The `$entity`, `$actio
 
 A couple of significant differences between APIv4 and APIv3 is that in v4 the `check_permissions` flag is set to true by default and on get actions especially but more generally the limit of 25 items returned that v3 has has been removed in favour of coders specifying the limit if they want to in their code calls.
 
+!!! note
```
Comment (MikeyMJCO) [inline docs/api/v4/usage.md]:
```suggestion
!!! info
```

File: docs/api/v4/usage.md [hunk 5]
```diff
@@ -11,6 +11,9 @@ Canonically, an API call is processed by the API kernel.  The `$entity`, `$actio
...
+   As of October 2019 Not all core entities have been added to apiv4. You should check the API explorer to see what entities are avaliable. If there is not one then please open a pull request to `civicrm-core` repository to add the entity in.
```
Comment (MikeyMJCO) [inline docs/api/v4/usage.md]:
```suggestion
   As of CiviCRM version 5,18 not all core entities have been added to APIv4. You should check the API explorer to see which entities are available. If the entity you require is not available then please open a pull request against the [`civicrm-core` repository](https://github.com/civicrm/civicrm-core) to add the entity in or open an (issue)[https://lab.civicrm.org/dev/core] and request that the entity is added.
```

**Comment by seamuslee001** [comment]
@MikeyMJCO changes incoporated

**Pull request merged**